### ISSUE TITLE
wbemcli docs/wbemcli.help.txt incomplete

### ISSUE THREAD

**Issue by KSchopmeyer** [pywbem/pywbem#1205]
I don't understand why but the help documentation for wbemcli docs.wbemcli.help.txt is incomplete and leaves several of the help items including the following items:

```

  -V, --version         Display pywbem version and exit.
  --statistics          Enable gathering of statistics on operations.
  -l log_spec[,logspec], --log log_spec[,logspec]
                        Log_spec defines characteristics of the various named
                        loggers. It is the form:
                         COMP=[DEST[:DETAIL]] where:
                           COMP:   Logger component name:[api|http|all].
                                   (Default=all)
                           DEST:   Destination for component:[file|stderr].
                                   (Default=file)
                           DETAIL: Detail Level to log: [all|paths|summary] or
                                   an integer that defines the maximum length of
                                   of each log record.
                                   (Default=all)

```

Strange sinte it is dated with todays date.

**Comment by andy-maier** [comment]
Fixed in PR #1208.

**Comment by andy-maier** [comment]
The fix for this issue has also been rolled back into 0.12.2.
